src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/ExtractToCodeBehindCodeActionProvider.cs

```diff
@@ -41,6 +42,12 @@ override public Task<CommandOrCodeActionContainer> ProvideAsync(RazorCodeActionC
             }
 
             var owner = syntaxTree.Root.LocateOwner(change);
+            if (owner == null)
```

Comment:
Null check for consistency with other locations (this was the only "breaking" `owner` null check, the rest are guarded).